src/lib/pages/backoffice/Item/ItemDetails/ItemHeader.js

```diff
@@ -8,9 +8,37 @@ import { BackOfficeRoutes } from '@routes/urls';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import { Header } from 'semantic-ui-react';
+import { Header, Label } from 'semantic-ui-react';
+import { invenioConfig } from '@config';
 
 export class ItemHeader extends Component {
+  statusLabel = () => {
+    const { data } = this.props;
+    let status = {};
+    if (
+      invenioConfig.ITEMS.canCirculateStatuses.includes(data.metadata.status)
+    ) {
+      if (
+        !invenioConfig.CIRCULATION.loanActiveStates.includes(
+          data.metadata.circulation.state
+        )
+      ) {
+        status = {};
+        status.text = 'On shelf';
+        status.styleClass = 'item-on-shelf-status';
+      } else {
+        status = invenioConfig.CIRCULATION.statuses.find(
+          x => x.value === data.metadata.circulation.state
+        );
+      }
+    } else {
+      status = invenioConfig.ITEMS.statuses.find(
+        x => x.value === data.metadata.status
+      );
+    }
+    return <Label className={status.styleClass}>{status.text}</Label>;
```

Comment:
after discussions with @kprzerwa too, my thinking was that I would not add a configuration `styleClass` in the `config.js` that is for a specific component (the label), because if in the future we have to add a new component that will need different classes, then we will start adding them to the config.

I would do instead:
```
    return <Label className=`item-status{lowercaseDashed(status.value)}`>{status.text}</Label>;
```
so that you will have in the end:
```
<Label className='item-status-for-reference-only'>For reference only</Label>;
<Label className='item-status-in-binding'>In binding</Label>;
```

and then only create all the CSS classes as you have already done:
```
.item-status-for-reference-only.label {
    background-color: #6435c9 !important;
    border-color: #6435c9 !important;
    color: #ffffff !important;
  }
```

What do you think?